sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -120,15 +117,15 @@
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PollerFlux<OperationResult, List<RecognizedForm>>
-        beginRecognizeCustomFormsFromUrl(String formUrl, String modelId, boolean includeTextDetails,
-        Duration pollInterval) {
-        final Duration interval = pollInterval != null ? pollInterval : DEFAULT_DURATION;
+        beginRecognizeCustomFormsFromUrl(RecognizeCustomFormsOptions recognizeCustomFormsOptions) {
```

Comment:
This method can be removed now as there is [beginRecognizeCustomForms(RecognizeCustomFormsOptions recognizeCustomFormsOptions)](https://github.com/Azure/azure-sdk-for-java/pull/12111/files#diff-f3e13c2bd95cecb8a90329a3c1e1f6dbR181).

Reply:
Updated it to be `package-private`.